test/StoragePickersTests/StoragePickersTests.cpp

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "AssemblyInfo.h"
+
+#include <FrameworkUdk/Containment.h>
+#include <winrt/Microsoft.Windows.Storage.Pickers.h>
+#include <TerminalVelocityFeatures-StoragePickers.h>
+
+namespace TB = ::Test::Bootstrap;
+namespace TP = ::Test::Packages;
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+namespace Test::StoragePickersTests
+{
+    class StoragePickersTests
+    {
+    public:
+        BEGIN_TEST_CLASS(StoragePickersTests)
+            TEST_CLASS_PROPERTY(L"ThreadingModel", L"MTA") // MTA is required for ::Test::Bootstrap::SetupPackages()
+            TEST_CLASS_PROPERTY(L"RunFixtureAs:Class", L"RestrictedUser")
+            //TEST_CLASS_PROPERTY(L"RunFixtureAs:Class", L"UAP")
+            //TEST_CLASS_PROPERTY(L"RunAs", L"UAP")
+
+            TEST_CLASS_PROPERTY(L"IsolationLevel", L"Method") // each test sets its own CompatibilityOptions
```

Comment:
>each test sets its own CompatibilityOptions [](http://example.com/codeflow?start=65&length=43)

Bad copy/paste. You could probably remove this line entirely, since I think method-level isolation should not be required by these tests.

Reply:
Thank you for your suggestion, fixed.